src/components/templates/Audience/Audience.tsx

```diff
@@ -361,13 +376,14 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
     return (
       <>
         <div
-          className="audience-container"
+          className={containerClasses}
           style={{
             backgroundImage: venue.mapBackgroundImageUrl
               ? `url(${venue.mapBackgroundImageUrl})`
               : undefined,
           }}
```

Comment:
if you use `useCss`, please remove inline styles then.

Reply:
removed b396ea2